app/components/avo/fields/common/stars_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="rating-group" data-component="<%= component_name %>"> 
+  <% (1..@max).each do |i| %>
+    <% star_classes = "star #{'filled' if i <= (@value || 0)} inline-block mr-0.5" %>
+    <%= helpers.svg('heroicons/solid/star', class: star_classes) %>
```

Comment:
Let's use the `class_names` helper for better readability, example: https://github.com/avo-hq/avo/blob/4-dev/app/components/avo/row_selector_component.html.erb#L8